models/edr/data_monitoring/data_monitoring/data_monitoring_metrics.sql

```diff
@@ -4,8 +4,10 @@
     unique_key='id',
     on_schema_change='append_new_columns',
     full_refresh=elementary.get_config_var('elementary_full_refresh'),
-    meta={"timestamp_column": "updated_at"}
+    meta={"timestamp_column": "updated_at"},
+    table_type="iceberg",
```

Comment:
Is there better way to provide connector specific config properties for some models?

Reply:
Nothing special, I guess this should work:
```
table_type="iceberg" if target.type = 'athena' else none
```

Reply:
Okey, I thought maybe there is some way to avoid having "table_type=None" for other connectors (e.g. to avoid possible conflicts). As an example, dbt_columns.sql has `materialized=elementary.get_dbt_columns_materialization(),` to have connector specific value, but here it should work for a set of properties, something like `kwargs`.